aten/src/ATen/cpu/vec256/functional.h

```diff
@@ -231,4 +231,33 @@ inline void map3(
   }
 }
 
+template <typename scalar_t, typename Op>
+inline void map4(
```

Comment:
Unused?

Reply:
used in [here](https://github.com/pytorch/pytorch/pull/48919/files#diff-e39a21a7125ac201b766a585b57ebf8429a7ac28cd723b09930aceb198fd25b0R510), the kernel file is folded by github since it is a bit large.